vm/src/stdlib/thread.rs

```diff
@@ -38,7 +36,7 @@ fn get_ident(_vm: &VirtualMachine) -> u32 {
 }
 
 fn allocate_lock(vm: &VirtualMachine) -> PyResult {
-    let module = import::import_module(vm, PathBuf::default(), "_thread")?;
+    let module = vm.import("_thread", &vm.ctx.new_tuple(vec![]), 0)?;
     let lock_class = vm.get_attribute(module.clone(), "RLock")?;
```

Comment:
Please replace this code by `let lock_class = vm.class("_thread", "RLock")`. This will call `vm.import`

Reply:
Done